### PR TITLE
chore: (QTM-411) Remove Quantum-ui

### PR DESCRIPTION
**Pull request by douglasjr97**
## Description
https://jirasoftware.catho.com.br/browse/QTM-411

## Review guide
- [ ] Code review
- [ ] Documentation

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import 'loki/configure-react';
-import { UiStyle } from '@catho/quantum-storybook-ui';
```
Comment (tcelestino) [inline .storybook/preview.js:2]: What do you think to remove import on file `scripts/templates/component-story.js`?

**Review by tcelestino — CHANGES_REQUESTED** [review]
suggests to improvements

File: .storybook/regression-test/preview.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
 import 'loki/configure-react';
-import { UiStyle } from '@catho/quantum-storybook-ui';
```
[the remark this answers is not in the document]
Comment (rapahaeru) [reply .storybook/regression-test/preview.js:2]: i agree

File: .storybook/preview.js (hunk 1 above)
Comment (rapahaeru) [reply .storybook/preview.js:2]: Its really should be removed...

Comment (rapahaeru) [reply .storybook/preview.js:2]: Its really should be removed...

File: scripts/create-component.js [hunk 3]
```diff
@@ -82,17 +81,6 @@ try {
     );
   });
 
-  const generatedComponentStoryTemplates = componentStoryTemplates(
```
Comment (rapahaeru) [inline scripts/create-component.js:85]: This code is used to generate a new component By `yarn generate <componentName>`, more info here: https://github.com/catho/quantum/blob/master/CONTRIBUTING.MD

**Review by rapahaeru — DISMISSED** [review]

File: scripts/templates/component-story.js [hunk 4]
```diff
@@ -51,6 +64,6 @@ const Template = args => <${componentName} {...args} />;
 
 export const Default = Template.bind({})
     `,
-    name: `${componentName}.regression-test.story.jsx`,
+    name: `${componentName}.stories.jsx`,
```
Comment (MarcosViniciusPC) [inline scripts/templates/component-story.js:67]: Default export between lines 58 - 61 of this file is not required

File: scripts/templates/component-story.js [hunk 5]
```diff
@@ -1,41 +1,54 @@
...
+
+<Meta title="${componentName}" component={${componentName}}  />
+
+<Header title="${componentName}">
+      Some description for component.
+</Header>
+
+- [Importing and usage](#importing)
+- [API](#api)
+- [Examples](#examples)
```
Comment (MarcosViniciusPC) [inline scripts/templates/component-story.js:21]: There isn't an examples section further down, what do you think about removing that link?

File: scripts/templates/component-story.js [hunk 6]
```diff
@@ -1,41 +1,54 @@
...
+
+<Canvas>
+  <Story story={Default} />
+</Canvas>
+
+## <a id="api"></a>API
+
+<ArgsTable of={${componentName}} />
+    
+
```
Comment (MarcosViniciusPC) [inline scripts/templates/component-story.js:33]:
```suggestion
```

File: scripts/templates/component-story.js [hunk 7]
```diff
@@ -1,41 +1,54 @@
 module.exports = componentName => [
+  {
+    content: `
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
```
Comment (MarcosViniciusPC) [inline scripts/templates/component-story.js:4]:
```suggestion
    content: `// Generated with scripts/create-componnent.js
import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
```

**Review by MarcosViniciusPC — CHANGES_REQUESTED** [review]
[ ] Code Review
[ ] Documentation

File: scripts/templates/component-story.js [hunk 8]
```diff
@@ -1,44 +1,39 @@
...
-    additionalTabs={TabExample}
-  />
-)
     `,
-    name: `${componentName}.story.jsx`,
+    name: `${componentName}.story.mdx`,
   },
   {
-    content: `// Generated with scripts/create-componnent.js
+    content: `
```
Comment (MarcosViniciusPC) [inline scripts/templates/component-story.js:36]: Why removed?

File: scripts/templates/component-story.js [hunk 9]
```diff
@@ -1,44 +1,39 @@
...
   {
-    content: `// Generated with scripts/create-componnent.js
-import {
-  AutoExample,
-  Tab,
-  StoryContainer,
-  Title,
-  Example,
-} from '@catho/quantum-storybook-ui';
+    content: `
```
Comment (MarcosViniciusPC) [inline scripts/templates/component-story.js:3]: Why did you remove the "// Generated with scripts/create-componnent.js" instead of adding it?

**Review by MarcosViniciusPC — CHANGES_REQUESTED** [review]
[x] Code review
[ ] Documentation

I missed the point mentioned earlier regarding the component import example in the documentation

File: scripts/templates/component-story.js [hunk 10]
```diff
@@ -21,6 +24,10 @@ import {
 
 ## <a id="importing"></a>Importing and usage
 
+${crasis}js
+import { ${componentName} } from '@catho/quantum';
+${crasis}
```
Comment (MarcosViniciusPC) [inline scripts/templates/component-story.js:29]: There's a way to improve this here with escape characters

**Review by MarcosViniciusPC — CHANGES_REQUESTED** [review]
[ ] Code Review
[ ] Documentation

**Review by MarcosViniciusPC — APPROVED** [review]
[x] Code review
[x] Documentation

**Review by tcelestino — APPROVED** [review]
- [x] Code review
- [x] Documentation
